### ISSUE TITLE
Admin/API: Fetch data sources

### ISSUE THREAD

**Issue by k8hughes** [devinit/ddw-analyst-ui#20]

**Comment by edwinmp** [comment]
Hi @Napho @akmiller01 for this, I'm going to need something similar to what's returned here http://212.111.41.68:8000/all_tables, only with more info. I think this is where you'll need to start thinking about how you want to structure the metadata.
What I'll need from this endpoint (keeping in mind issue #22) is: -

- **tableId** (I imagine this will take the shape of `[schema].[table_name]` - don't think we need to split them into table schema & table name, choice is yours).
- **title/name** (think you should find a way to auto generate the default value for this from the table_name - without the underscores)
- **columns**  - an array of objects, each object with a column name plus some metadata i.e id, displayName, metadata
- **metadata** - however you imagine this. OECD stats seem to split it into abstract, sources, date (updated, created), other
- **published** - true/false

I don't mind having all this returned in one end point, shouldn't be heavy, and ofc these are just my thoughts on this, so it maybe bigger than it seems or take a different shape. You get the gist though :)

**Comment by edwinmp** [comment]
@akmiller01 @Napho  are you now certain of what shape this data is going to take? Don't intend to wait on the API; I'm going to mock it, and want to be accurate.

**Comment by akmiller01** [comment]
@edwinmp , you should be able to run `python manage.py loaddata test_data` to get the test data in the form that @Napho was intending. All the columns can be intuited from here https://github.com/devinit/ddw-analyst-ui/blob/feature/models-clean/core/models.py#L87-L132

I think those three models should constitute the fields you were imagining.

**Comment by akmiller01** [comment]
https://github.com/devinit/ddw-analyst-ui/pull/59
